submissions/NJul/html-css-popup/style.css

```
@@ -0,0 +1,420 @@
+*,
+::after,
+::before {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  height: 100%;
+
+  font-family: 'Noto Sans JP', sans-serif;
+}
+
+.background {
+  margin: 0;
+  padding: 0;
+  height: auto;
+  width: 100%;
+  min-height: 100vh;
+
+  overflow-x: hidden;
+
+  background-image: url(./img/background-cat-small.jpg);
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+  background-color: #464646;
+}
+
+.flex-horizontal-right {
+  display: flex;
+  width: 100%;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+a {
+  text-decoration: none;
+  border: 1px solid transparent;
+  transition: all .25s ease-in-out;
+}
+
+img {
+  max-width: 100%;
+}
+
+/* Menu */
+
+.navbar {
+  width: 100%;
+  margin: auto;
+  background: rgb(255, 255, 255, 0.8);
+  background: linear-gradient(-690deg, rgba(255, 255, 255, 0) 0%, rgb(255, 255, 255, 0.8) 80%);
+  min-height: 60px;
+  position: relative;
+}
+
+/* For positioning sub menus */
+.navbar li {
```

It's better to avoid using tags as selectors.
```suggestion
.navbar .navbar__item {
```
For better understanding:
https://frontstuff.io/you-need-to-stop-targeting-tags-in-css